backend/game/cards/keep_card.py

```diff
@@ -7,11 +7,11 @@ def __init__(self, name, cost, effect):
         super().__init__(name, cost, effect)
 
     def immediate_effect(self, player_that_bought_the_card, other_players):
-        player_that_bought_the_card.add_card(self)
+        pass
 
     @abstractmethod
     def special_effect(self, player_that_bought_the_card, other_players):
         raise NotImplementedError
 
     def discard_effect(self, player_that_bought_the_card, other_players):
-        player_that_bought_the_card.remove_card(self)
\ No newline at end of file
+        pass
```

Comment:
removed internal adding and removing of card to player... thinking higher order centralized place should add card to players hand... but could be wrong.

Can put back if you all prefer.

Reply:
Gerardo and I agree with these being removed, however are planning to handle this in a future refactor